src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyExecutionManager.cs

```diff
@@ -105,7 +109,7 @@ public virtual int StartTestRun(TestRunCriteria testRunCriteria, ITestRunEventsH
                     this.InitializeExtensions(testSources);
                 }
 
-                this.SetupChannel(testRunCriteria.Sources);
+                this.SetupChannel(testRunCriteria.Sources, this.cancellationTokenSource.Token);
```

Comment:
Should we consider defining this CancellationTokenSource in ProxyOperationManager?